Trell.Engine/RuntimeApis/BrowserApi.cs

```diff
@@ -351,18 +351,22 @@ string TrimLogMessage(string msg) {
             }
 
             public void LogInformation(string msg) {
+                ThrowIfCanceled();
```

Comment:
```suggestion
                this.Context.CancellationToken.ThrowIfCancellationRequested();
```

SQLiteApi uses this mechanism.

Reply:
Resolved in commit 4658af4